iot-hub/Samples/device/DeviceStreamingSample/DeviceClientStreamingSample.csproj

```diff
@@ -16,6 +16,6 @@
   </ItemGroup>
   
   <ItemGroup>
-	<PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.29.0-preview-*" />
+	<PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.32.0-preview-001" />
```

Comment:
Upgraded sample to use the latest version where this sample will work for net 5.0 as well